### ISSUE TITLE
Datatable - attribute output - numberOfRowsSelected not updated in real time

### ISSUE THREAD

**Issue by cadillacben** [alexed1/LightningFlowComponents#695]
This worked (and still does) correctly in 3.0.9, FYI.

### Steps to reproduce

1. add datatable to a screen
2. set some component to be visible depending on the numberOfRowsSelected value being greater than 0
3. run the flow in debug (same for other ways)
4. select one row of data

### Expected behaviour
The other component should become visible when any row is selected

### Actual behaviour
Nothing happens

I'm suspect of line1181 being commented out in the datatable.js, but I'm no developer.
// this.dispatchEvent(new FlowAttributeChangeEvent('numberOfRowsSelected', this.numberOfRowsSelected));

The output does get updated after clicking next and going to the next screen.

**Comment by ericrsmith35** [comment]
Thanks for the feedback.  I think you are correct.  I'll address this in the next release.

**Comment by cadillacben** [comment]
Thank you for creating this and making it available!

I don't know if this is a bug or just how it works, but the table displays differently if there is only one row.  To get around the number of rows selected issue I decided to just show an error button/message if a row was selected which proved tricky until I realized this happens.
![multiRow](https://user-images.githubusercontent.com/17458691/115630694-71bd5f00-a2d2-11eb-8f70-9f7d838f9d77.png)
![singleRow](https://user-images.githubusercontent.com/17458691/115630696-7255f580-a2d2-11eb-8e5f-f4fe0a5a7896.png)
